src/server/rdb_save.cc

```diff
@@ -890,6 +890,11 @@ RdbSaver::RdbSaver(::io::Sink* sink, SaveMode save_mode, bool align_writes) {
 RdbSaver::~RdbSaver() {
 }
 
+error_code RdbSaver::CloseImmediately(::io::Sink* sink) {
```

Comment:
is it like a temporary code? EOF token is announced before full sync starts and you loose this info this way.